qemu/tests/cfg/hotplug_mem_negative.cfg

```diff
@@ -20,7 +25,7 @@
             size_mem = 0G
             keywords = "doesn\'t take value \'0\'"
         - invalid_backend_ram_option:
-            policy_mem = bind
+            policy_mem_plug1 = bind
```

Comment:
@huangyum  This line change the policy of plug1 from `default` to `bind` ?   Could you explain this ?

Reply:
No, policy_mem will set all memdev to use bind. With this change, it only restrict memdev plug1.

Reply:
Thanks